# message to set a shader param value on a Gazebo Visual

int8 SHADER_TYPE_VERTEX   = 0
int8 SHADER_TYPE_FRAGMENT = 1
int8 NUM_SHADER_TYPES     = 2

int8   shaderType
string paramName
string paramValue
